hook/DestIPFoundHook.js

```diff
@@ -251,25 +265,25 @@ class DestIPFoundHook extends Hook {
     let forceRedisUpdate = options.forceUpdate;
 
     return (async() => {
+      let sslInfo = await intelTool.getSSLCertificate(ip);
+      let dnsInfo = await intelTool.getDNS(ip);
```

Comment:
sslInfo and dnsInfo are not guaranteed to be the exact domain this device is really accessing. 

this could potentially reduce the number of entry to be added to category domain

Reply:
Current implementation uses these two info to determine domain... I just move this piece of code ahead of cache validation...